Flick/Cells/EditCollaboratorTableViewCell.swift

```diff
@@ -84,8 +85,8 @@ class EditCollaboratorTableViewCell: UITableViewCell {
     func configureCollaborator(for user: UserProfile, isOwner: Bool) {
         self.user = user
         nameLabel.text = user.name
-        if let imageUrl = URL(string: user.profilePic?.assetUrls.original ?? "") {
-            userImageView.kf.setImage(with: imageUrl)
+        if let profilePic = user.profilePic {
+            userImageView.kf.setImage(with: Base64ImageDataProvider(base64String: profilePic, cacheKey: "collaboratorProfilePicture"))
```

Comment:
looks like the cacheKey need to be different for each image based on here:
https://github.com/onevcat/Kingfisher/blob/175eeb4618b0a6ef4d69a7409b6a74ddd235a093/Sources/General/ImageSource/ImageDataProvider.swift#L112

so i think we can prob add like the user id to the cacheKey?

Reply:
oh ya haha I realized this when I was doing my clean up so this will be changed in the next PR!